x-pack/plugins/security_solution/public/overview/pages/translations.ts

```diff
@@ -29,6 +29,13 @@ export const DATA_QUALITY_TITLE = i18n.translate(
   }
 );
 
+export const DATE_PICKER_TOOLTIP = i18n.translate(
+  'xpack.securitySolution.dataQualityDashboard.datePicker.tooltip',
+  {
+    defaultMessage: `Conducting data quality checks is possible solely for the data present within the Ingest and Search Boost window (last 7 days).`,
```

Comment:
@dhru42 would you be willing to follow up re: updating this tooltip, per our offline discussion with @angorayc ?

Reply:
Open an issue to follow this up: https://github.com/elastic/kibana/issues/164672